packages/suite-data/files/translations/en-US.json

```diff
@@ -280,6 +280,8 @@
   "TR_ACQUIRE_DEVICE": "Use Trezor here",
   "TR_ACQUIRE_DEVICE_TITLE": "Another session is running",
```

Comment:
no need to modify this file

Reply:
675de0fdfefe6b37e5ed82c54dbf96fada24a91e